src/commands/mod.rs

```diff
@@ -178,6 +179,12 @@ fn app() -> Command<'static> {
                 .global(true)
                 .help("Do not print any output while preserving correct exit code. This flag is currently implemented only for selected subcommands."),
         )
+        .arg(
+          Arg::new("allow_failure")
```

Comment:
To keep consistent with the other commands, use - instead of _ to split words:
```suggestion
          Arg::new("allow-failure")
```